rm/rm.h

```diff
@@ -315,6 +313,10 @@ struct m0_rm_resource {
 	 * with which local owners of credits to this resource communicates.
 	 */
 	struct m0_tl                     r_remote;
```

Comment:
I propose to change this to plural form: r_remotes
Because this is a list of remote owners.

Reply:
Makes sense, yes. Will do, thanks.